lang/cs/Org.Apache.REEF.Client/Common/DriverFolderPreparationHelper.cs

```diff
@@ -38,13 +38,23 @@ internal sealed class DriverFolderPreparationHelper
     {
         private const string DLLFileNameExtension = ".dll";
         private const string EXEFileNameExtension = ".exe";
+        private const string BridgeExe = "Org.Apache.REEF.Bridge.exe";
+        private const string BridgeExeConfig = "Org.Apache.REEF.Bridge.exe.config";
+        private readonly string DefaultDriverConfigurationFileContents =
+        @"<configuration>" +
+        @"  <runtime>" +
+        @"    <assemblyBinding xmlns=""urn:schemas-microsoft-com:asm.v1"">" +
+        @"      <probing privatePath=""reef/local;reef/global;local;global""/>" +
+        @"    </assemblyBinding>" +
+        @"  </runtime>" +
+        @"</configuration>";
+
         
         private readonly static Tuple<string, string>[] clientFileResources = new Tuple<string, string>[]
         {
             new Tuple<string, string>("ClientJarFullName", "reef_bridge_client"),
             new Tuple<string, string>("DriverJarFullName", "reef_bridge_driver"),
-            // enable with next pull request
-            // new Tuple<string, string>("ClrDriverFullName", "reef_clrdriver"),
+            new Tuple<string, string>("ClrDriverFullName", "reef_clrdriver"),
```

Comment:
What do these strings signify? If they are magic strings, maybe they should be documented `consts`?